dot/telemetry/afg_authority_set.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package telemetry
+
+// afgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
+// meant to be sent when authority set changes (generally when a round is initiated)
+type afgAuthoritySetTM struct {
+	//nolint
+	// finalized hash and finalized number are sent by substrate, but not read by
+	// substrate telemetry
+	// FinalizedHash   *common.Hash `json:"hash"`
+	// FinalizedNumber string       `json:"number"`
```

Comment:
how much extra effort would it be to add these?

Reply:
None, hash and number can be achieved by `s.head.Hash()` and `s.head.Number.String()`.

I did not add them because telemetry server does not read those data. substrate also doesn't send these all the time. There is also another telemetry message through which we are sending these info `afg.finalized_blocks_up_to`

Reply:
yeah either just add them, or remove this